spec/models/comment_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+RSpec.describe comment, type: :model do
+  context 'validations' do
+    it "checks length of comment" do
+      comment = Comment.new(commenter: "gaurav", body: "hello world")
+      expect(comment).to_not be_valid
```

Comment:
I don't see any validations in comment model why comment object should be invalid here?